### PR TITLE
Added base quick query project

### PR DESCRIPTION
**Pull request by gapra-msft**

### CODE REVIEW

File: sdk/storage/azure-storage-quickquery/pom.xml [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+      </releases>
+      <snapshots>
+        <enabled>false</enabled>
+      </snapshots>
+    </pluginRepository>
+  </pluginRepositories>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.azure</groupId>
```
Comment (rickle-msft) [inline sdk/storage/azure-storage-quickquery/pom.xml]: We shouldn't have a core dependency. It should pull it from blobs

File: sdk/storage/azure-storage-quickquery/pom.xml [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+      <artifactId>azure-core-http-netty</artifactId>
+      <version>1.3.0</version> <!-- {x-version-update;com.azure:azure-core-http-netty;dependency} -->
+    </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-storage-blob</artifactId>
+      <version>12.5.0-beta.1</version> <!-- {x-version-update;com.azure:azure-storage-blob;current} -->
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
```
Comment (rickle-msft) [inline sdk/storage/azure-storage-quickquery/pom.xml]: I believe this as well can be pulled from a parent.

File: sdk/storage/azure-storage-quickquery/pom.xml (hunk 1 above)
Comment (gapra-msft) [reply sdk/storage/azure-storage-quickquery/pom.xml]: done

File: sdk/storage/azure-storage-quickquery/pom.xml (hunk 2 above)
Comment (gapra-msft) [reply sdk/storage/azure-storage-quickquery/pom.xml]: done

**Comment by rickle-msft** [comment]
@JimSuplizio Can you take a look at this as well, please?

**Comment by gapra-msft** [comment]
All checks have passed but they have not reported status. Seems like an issue with GitHub so I'm just going to merge this.

**Pull request merged**